checks/org/check_test.go

```diff
@@ -0,0 +1,81 @@
+package org
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/Shopify/voucher"
+	r "github.com/Shopify/voucher/repository"
+	vtesting "github.com/Shopify/voucher/testing"
+)
+
+func TestOrgCheck(t *testing.T) {
+	c := context.Background()
+
+	repoClient, _ := vtesting.NewClient()
+	commits := map[string]r.Commit{
+		"cdef0987": {URL: "https://github.com/Shopify/app/commit/cdef0987", Checks: nil, Status: "good", IsSigned: true,
+			AssociatedPullRequests: nil},
+		"efgh6543": {URL: "https://github.com/Shopify/app/commit/efgh6543", Checks: nil, Status: "great", IsSigned: false,
+			AssociatedPullRequests: []r.PullRequest{{BaseBranchName: "base", HeadBranchName: "master", IsMerged: false},
+				{BaseBranchName: "branch-base", HeadBranchName: "test", IsMerged: true}}},
+	}
+	branches := map[string]r.Branch{
+		"test":   {Name: "test", CommitRefs: []r.CommitRef{{URL: "https://github.com/Shopify/app/commit/efgh6543"}}},
+		"master": {Name: "master", CommitRefs: []r.CommitRef{{URL: "https://github.com/Shopify/app/commit/cdef0987"}}},
+	}
+	repoClient.AddRepository(r.CreateNewOrganization("Shopify", "https://github.com/Shopify"), "Repo_Shopify", commits, branches)
```

Comment:
I just realized this is in an issue since we changed `CreateNewOrganization` to `NewOrganization` in the other PR, could you update this to:

```suggestion
	repoClient.AddRepository(r.NewOrganization("Shopify", "https://github.com/Shopify"), "Repo_Shopify", commits, branches)
```